native_client/deepspeech.cc

```diff
@@ -42,6 +44,7 @@ Model::Model(const char* aModelPath, int aNCep, int aNContext,
              const char* aAlphabetConfigPath, int aBeamWidth)
 {
   mPriv             = new Private;
+  mPriv->mmap_env   = new MemmappedEnv(Env::Default());
```

Comment:
This is only used in this constructor, so there's no need to put it in `Private`.

Reply:
We actually need to hold the pointer for the duration of `Model` :)